### DISCUSSION TITLE
#include glsl CompileShaderInclude

### DISCUSSION THREAD

**Discussion by BuslikDrev** [opentk/opentk#1693] [General]
Has anyone tried connecting an additional shader via OpenTK.Graphics.OpenGL4.GL.Arb.CompileShaderInclude()?

**Comment by NogginBops** [comment]
You'll probably have better luck asking in our discord https://discord.gg/6HqD48s
There it's a lot easier to get answers to questions like this. I think there are a few people there that have used it.
